client/components/signup-form/index.jsx

```diff
@@ -591,6 +595,20 @@ class SignupForm extends Component {
 			<div className={ classNames( 'signup-form', this.props.className ) }>
 				{ this.getNotice() }
 
+				{ this.props.isSocialSignupEnabled &&
+					this.props.isSocialFirst && (
```

Comment:
We could save an indentation level by putting this condition on one line:

```javascript
{ this.props.isSocialSignupEnabled && this.props.isSocialFirst && (
```

Reply:
This was automatically indented by our "beautifier"